### DISCUSSION TITLE
Abrir Lives directamente en la App de Teams [Linux]

### DISCUSSION THREAD

**Discussion by david-ceja** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#9] [Q&A]
Hola hermanxs.
De casualidad alguien podrá orientarme respecto a cómo abrir los Lives grabados en la App de Teams?
Cada que doy clic a un live dentro de Teams (ya sea el enlace llamado "Soy Enlace" o "Microsoft Teams"), se abre una pestaña nueva en mi navegador Firefox, indicándome si quiero abrir el enlace en la App de Teams (un tanto redundante a mi parecer, siendo que desde ahí fue que presioné el enlace), o si deseo ver el live vía web en esa pestaña.
![Screenshot from 2022-02-17 04-03-25](https://user-images.githubusercontent.com/98674217/154454353-bb5a24e5-7056-4f7c-ab02-ef390d523a8a.png)

Considero que sería más óptimo vía la App de Teams, pero la cuestión es que siempre que elijo esa opción, Teams se queda cargando unos 15 segundos y luego me salta el error "Joining a live event with external encoder is not enabled"
![Screenshot from 2022-02-17 04-04-00](https://user-images.githubusercontent.com/98674217/154454388-f30ab47e-210f-4164-ae79-83dc133c9012.png)

Así que tengo que optar por ver el live en el navegador web, aunque tiene la desventaja de que tarda muchísimo en cargar la plataforma Teams via web, y a veces, después de finalmente cargar todo ese tiempo, la plataforma me arroja un error (no recuerdo exactamente cuál, pero empieza por "Oopsss", aludiendo a que no se ha podido encontrar o cargar el material.
Y tengo que volver a cerrar la pestaña y abrir el enlace, elegir que quiero verlo via web y volver a esperar otro minuto entero para que la plataforma cargue vía web, y pues cruzar los dedos para que esta vez sí muestre el live.

Quiero pensar que se debe a  que me falta configurar algo que se me está escapando, y no debido a alguna incompatibilidad de la App en Linux =S
Durante el curso de Python, me fue prácticamente imposible ver el último live de Dudas sobre los Entregables, por que siempre me daba error la plataforma vía web. Igual no era material indispensable, así que al final pude sortearlo, pero quisiera evitar esta situación e iniciar con el pie derecho Front-End.

De antemano una disculpa si esto ya había tratado en la sección de Discusiones del curso de Python, pero como estaba presionado por terminar antes del tiempo límite que tenía la idea era mañana viernes, no tuve oportunidad de integrarme a él.
Cualquier idea es muy bienvenida.
Les agradezco mucho sinceramente su tiempo y atenciones.
Mucho éxito. Y ahora que ya me emparejé, espero toperles por aquí de vuelta pronto (no solo para preguntar, sino para ver en qué podemos apoyar n_n )

**Comment by Shellcode-KR** [comment]
lo que te podría recomendar es que intentes abrirla en el navegador chromium

**Comment by doguedogue** [comment]
Las videos de teams primeramente se alcanzan como url, es decir para ver en el navegador, si tienes instalado el teams como app debería abrírtelo sin problemas. Pero debería ser igual como se ve en el navegador que en el teams (app).

Si ya te abre la app es porque la tienes correctamente configurada, quizás solo tienes que configurar algo extra.

Igual sigue esta guía a lo mejor te ayuda: https://docs.microsoft.com/en-us/answers/questions/5713/how-to-join-teams-meetings-from-linux-client.html

**Comment by pdroivan** [comment]
Usas tu app teams para otra cuenta? Trabajo/Escuela, en mi caso yo si, y no me permite si quiera iniciar sesión en la app con la cuenta de innovaccion, revisa si la versión de app es la que está especificada en el onboarding. Saludos

**Comment by david-ceja** [comment]
Muchas gracias por su apoyo hermanos.
No uso teams para otros fines, más que para este curso (nunca antes había usado la plataforma).
Pero ya me resigné a ver los lives grabados desde el navegador (por suerte ya no está fallando casi tanto la carga de los videos), pero les agradezco sus puntos de vista. 
A darle, que no es excusa n_n

**Comment by MauroMontan** [comment]
El problema puede ser originado por wayland, yo tuve algunos problemas con zoom y google meet que fueron originados por eso, lo que tienes que hacer es cerrar sesión y cambiar el perfil por Xorg (como se ve en esta imagen)

![image](https://user-images.githubusercontent.com/68311575/154872291-a3f31f64-9466-42a9-851c-c193b8b61d07.png)

espero que esto te ayude!
